wger/nutrition/templates/plan/view.html

```diff
@@ -144,7 +144,7 @@
         {% if is_owner %}
         <tr>
             <td colspan="6">
-            <a href="{% url 'nutrition:meal:add' plan.id %}"
+            <a href="{% url 'nutrition:meal_item:add-meal' plan.id %}"
```

Comment:
maybe you can let us know why the URL changed as @wekesa931  has stated

Reply:
> The previous process of adding a meal was:
> 
> 1. **`Add time`** - this would create a meal plan with an empty list of meals. The previous `url` pointed to this i.e `nutrition:meal:add`
> 2. **`Add meal`** - populate empty meal list with meal items. The `url` for this was `nutrition:meal_item:add`.
> 
> I combined these two `url`s to make `nutrition:meal_item:add-meal` so as to avoid having to go through the two steps. Now the time and meal are added at the same time.
> 
> The two original separate `url`s are still there. I did not remove them in case other portions of the code require them. I tested it to make sure it will not affect other areas of the code.